internal/builders/container/README.md

```diff
@@ -152,6 +153,115 @@ jobs:
       registry-password: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+#### With GCP Artifact Registry
+
+The following is an example of pushing an image to an [Artifact Registry](https://cloud.google.com/artifact-registry) in GCP and generating the provenance for that image. In order for you to run this example, you will need to have a [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) that enables you to exchange a GitHub token for access within GCP. If you have not yet created one or have not created a provider within your existing federation for GitHub, please review the following resources:
+
+- [Setting up Workload Identity Federation](https://github.com/google-github-actions/auth#setting-up-workload-identity-federation)
+- [Enabling keyless authentication from GitHub Actions](https://cloud.google.com/blog/products/identity-security/enabling-keyless-authentication-from-github-actions)
+
+Once you have a Workload Identity Federation with a GitHub provider, you're ready to begin implementing the GitHub Action below. 
+
+Friendly reminder to set the following environment variables in your GitHub settings:
+```bash
+  # EXAMPLE:
```

Comment:
nit: maybe provide a link to the GitHub documentation. Or maybe the original blog post. I don't think it's a well-known feature.